tools/scripts/build/build-mobile-curriculum.js

```diff
@@ -12,7 +16,14 @@ exports.buildMobileCurriculum = function buildMobileCurriculum(json) {
       key => key !== 'certifications'
     );
 
-    writeToFile('availableSuperblocks', { superblocks: superBlockKeys });
+    writeToFile('availableSuperblocks', {
+      // removing "/" as it will create an extra sub-path when accessed via an endpoint
+
+      superblocks: [
+        superBlockKeys.map(key => key.replace(/\//, '-')),
+        getSuperBlockNames()
```

Comment:
```suggestion
        getSuperBlockNames(superBlockKeys)
```
to keep the order consistent